hr_expense_invoice/models/hr_expense_sheet.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2015 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2017 Vicent Cubells <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, models
+from odoo.exceptions import UserError
+from odoo.tools import float_compare
+
+
+class HrExpenseSheet(models.Model):
+    _inherit = "hr.expense.sheet"
+
+    @api.multi
+    def action_sheet_move_create(self):
+        precision = self.env['decimal.precision'].precision_get('Account')
```

Comment:
Hello @kittiu 

Now this `decimal.precision` do no exists

Reply:
Fixed to 'Product Price'